proof/crefine/ARM/Tcb_C.thy

```diff
@@ -2020,7 +2019,7 @@ shows
                         apply (clarsimp simp: min_def iffD2 [OF mask_eq_iff_w2p] word_size
                                               word_less_nat_alt
                                       split: if_split_asm dest!: word_unat.Rep_inverse')
-                       apply simp
+                       apply (simp add: pred_conj_def)
```

Comment:
This is a case where the previous proof could automatically rewrite `pred_conj` due to the postcondition being eta-expanded.